docs/elasticsearch.md

```diff
@@ -0,0 +1,65 @@
+# `elasticsearch_json` and `elasticsearch_gc` Logging Receivers
+
+## Configuration
+
+To configure a receiver for your Elasticsearch JSON logs, specify the following fields:
+
+| Field                 | Default                       | Description |
+| ---                   | ---                           | ---         |
+| `type`                | required                      | Must be `elasticsearch_json`. |
+| `include_paths`       | `[/var/log/elasticsearch/*_server.json, /var/log/elasticsearch/*_deprecation.json, /var/log/elasticsearch/*_index_search_slowlog.json, /var/log/elasticsearch/*_index_indexing_slowlog.json, /var/log/elasticsearch/*_audit.json]` | The log files to read. |
+| `exclude_paths`       | `[]`                          | Log files to exclude (if `include_paths` contains a glob or directory). |
+
```

Comment:
Also the `wildcard_refresh_interval` config? sample: https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/apache#apache-logs

Same below for `Elasticsearch GC logs`.

Reply:
I've added these to the Elasticsearch docs.

Reply:
Thanks!